packages/amplify-cli-core/API.md

```diff
@@ -869,6 +930,16 @@ export function getGraphQLTransformerOpenSearchDocLink(version: number): string;
 // @public (undocumented)
 export function getGraphQLTransformerOpenSearchProductionDocLink(version: number): string;
 
+// @public (undocumented)
+export const getImportedAuthProperties: (context: $TSContext) => {
```

Comment:
Is cli-core right place for this ?

Reply:
See here for thoughts https://github.com/aws-amplify/amplify-cli/pull/12069#discussion_r1117375855